lua/render-markdown/handler/markdown.lua

```diff
@@ -374,7 +378,8 @@ function Handler:code_left_pad(code_block, add_background)
     end
     local padding = str.pad(code.left_pad)
     local highlight = add_background and code.highlight or 'Normal'
-    for row = code_block.start_row, code_block.end_row - 1 do
+    -- skip the first row, as it is already handled by 'language_pad'
+    for row = code_block.start_row + 1, code_block.end_row - 1 do
```

Comment:
Thank you for your prompt feedback.

From my testing, all is fine with your example.

From what I understand, the 'first line' is the one with markers, whether there's a language or not.

I'm on mobile right now, and might be missing something. I'll look further into it as soon as possible.

Reply:
In general you're right but we modify the state of `code_block` to handle different borders: https://github.com/MeanderingProgrammer/render-markdown.nvim/blob/main/lua/render-markdown/handler/markdown.lua#L338

Guessing you found this based on the update but just in case you were curious as to why.

Reply:
[c11e0f0](https://github.com/MeanderingProgrammer/render-markdown.nvim/pull/131/commits/c11e0f0a1c49749b2ffcdb8cad0674e21dc7beb2)

I used `code_block.language` as a check, it felt natural.